src/main/java/module/youth/YouthSkillInfo.java

```diff
@@ -193,6 +193,10 @@ public boolean isMaxReached() {
         return isMaxReached;
     }
 
+    public boolean isTrainingUsefull(){
+        return !isMaxReached() && (!this.isMaxAvailable() || this.getMax()>4);
+    }
```

Comment:
Pardon my ignorance, what is the logic here for a training to be deemed “useful?”

Reply:
this is part of my youth training potential calculation.

each skill of a youth player has an maximum value which is not known from the beginning, except for one skill mentioned by the scout. if this maximum value is reached, no further training of this skill is useful.

if this maximum is less than 5, i think training of this skill is not useful either.

Reply:
add a comment to that method